TESTING.md

```diff
@@ -149,13 +153,7 @@ a manual testing manual. Marked with a github issue label "manual testing".
         few seconds for the list of modules to be initialized. If the initialization takes more
         than 10&nbsp;seconds, consider it an error.</sub>
       </li>
-      <li>Select <em>GoodStuff</em> from the list and double click it.</li>
-      <li>
-        Ensure that <em>GoodStuff</em> and <em>O1Library</em> appear as loaded modules in the
-        project tree, and their contents match the image below:<br/>
-        <img src="images/30_module_loaded.png" alt="GoodStuff and O1Library contents" /><br/>
-        <sub>It shouldn't take longer than 5 seconds for modules to be imported.</sub>
-      </li>
+      <li>Select <em>GoodStuff</em> from the list and double click it to install the module.</li>
```

Comment:
I suggest that we do not re-ensure things that are already checked in other parts of the test procedure.

Reply:
I agree. another idea I had was to create some kind of a single longer script that would test all the features wo repeatings (thus, this will need to logically restructure things and thinking a bit on this).